### ISSUE TITLE
HTML code is lost in the Abstract-field (Layer tab)

### ISSUE THREAD

**Issue by guenterw** [3liz/lizmap-plugin#470]
## Description
In the Abstract field (Layer tab) the entered HTML code has disappeared the next time the plugin is called.
Basically the HTML code works on the WebClient

```html
<img src="https://................./Vorschaubild.jpg" width="150" height="159">
```

![image](https://user-images.githubusercontent.com/8373176/213478955-672840ab-fad5-445a-967d-25ec7de91121.png)

## Context
* QGIS version:  3.22.14
* Lizmap version (plugin): 3.10.0 
* OS:  Win10Pro

**Comment by josemvm** [comment]
duplicate? https://github.com/3liz/lizmap-plugin/issues/72

**Comment by guenterw** [comment]
yes, I was not aware of that.

I also don't quite understand the end of #72. Was the bug fixed then or is that a workaround?
Or is HTML supported in the field at all?

I was assuming a new bug, similar to https://github.com/3liz/lizmap-plugin/issues/464

**Comment by Gustry** [comment]
Fixed 2d2dd50
